src/server/auth/cmds/cmds.go

```diff
@@ -401,22 +403,35 @@ func ListAdminsCmd() *cobra.Command {
 // ModifyAdminsCmd returns a cobra command that modifies the set of current
 // cluster admins
 func ModifyAdminsCmd() *cobra.Command {
-	var add []string
-	var remove []string
+	var user string
+	var scopesStr []string
 	modifyAdmins := &cobra.Command{
-		Short: "Modify the current cluster admins",
-		Long: "Modify the current cluster admins. --add accepts a comma-" +
-			"separated list of users to grant admin status, and --remove accepts a " +
-			"comma-separated list of users to revoke admin status",
+		Short: "Set the admin scopes for a given user",
+		Long: "Set the cluster admin scopes for a given user." +
+			"--user is the user to modify, and --scopes is a comma-separated list of scopes." +
+			"An empty list removes all admin scopes for the user.",
 		Run: cmdutil.Run(func([]string) error {
 			c, err := client.NewOnUserMachine("user")
```

Comment:
IMO this should validate that `user` isn't empty (and perhaps that `scopes` isn't empty, though it would be reasonable in that case to simply do nothing)

Given that IIUC, both `user` and `scopes` would typically be set, it might be ergonomic to make them positional arguments and change the command to `modify-admin` (singular), so that invocation looks like `pachctl auth modify-admin alice super`, analogous to `pachctl auth set alice reader myrepo`.

(It also occurred to me that we could fold this into `pachctl auth set` and just have `cluster` take the place of the repo name, so `pachctl auth set alice super cluster`, but then we'd have to ban repos named `cluster`)